cfg/windows.cfg

```diff
@@ -3628,6 +3638,102 @@ HFONT CreateFont(
       <not-uninit/>
     </arg>
   </function>
+  <!--HMODULE WINAPI LoadLibrary(
+  _In_ LPCTSTR lpFileName);-->
+  <function name="LoadLibrary,LoadLibraryA,LoadLibraryW">
+    <noreturn>false</noreturn>
+    <returnValue type="HMODULE"/>
+    <arg nr="1">
+      <not-null/>
+      <not-uninit/>
+      <strz/>
+    </arg>
+  </function>
+  <!--HMODULE WINAPI LoadLibraryEx(
+  _In_       LPCTSTR lpFileName,
+  _Reserved_ HANDLE  hFile,
+  _In_       DWORD   dwFlags);-->
+  <function name="LoadLibraryEx,LoadLibraryExA,LoadLibraryExW">
+    <noreturn>false</noreturn>
+    <returnValue type="HMODULE"/>
+    <arg nr="1">
+      <not-uninit/>
+      <strz/>
+    </arg>
+    <arg nr="2">
+      <valid>0</valid>
```

Comment:
so this parameter must be 0?

Reply:
According to the [MSDN](https://msdn.microsoft.com/en-us/library/windows/desktop/ms684179(v=vs.85).aspx) this parameter is reserved and always must be NULL.
Since i found no way to tell cppcheck that it must be NULL, i have used the <valid>0</valid> entry. In my tests this works without problems. Is there a better way to check it or is it better to ignore it? What would you say?

Reply:
there is no better way. I just thought it was strange.